frontend/src/components/graduations/GraduationCard.tsx

```diff
@@ -0,0 +1,263 @@
+'use client';
+
+import { Graduation } from '@/database/graduation';
+import { formatRatingSystem } from '@/database/user';
+import CohortIcon from '@/scoreboard/CohortIcon';
+import { ChessDojoIcon } from '@/style/ChessDojoIcon';
+import { RatingSystemIcon } from '@/style/RatingSystemIcons';
+import ArrowDownwardIcon from '@mui/icons-material/ArrowDownward';
+import ArrowUpwardIcon from '@mui/icons-material/ArrowUpward';
+import { Box, Stack, Typography } from '@mui/material';
+import { ReactNode, useMemo } from 'react';
+import { AxisOptions, Chart, UserSerie } from 'react-charts';
+
+interface Datum {
+    date: Date;
+    rating: number;
+}
+
+export const primaryAxis: AxisOptions<Datum> = {
+    scaleType: 'time',
+    getValue: (datum) => datum.date,
+};
+
+export const secondaryAxes: AxisOptions<Datum>[] = [
+    {
+        scaleType: 'linear',
+        getValue: (datum) => datum.rating,
+        formatters: {
+            scale: (value) => `${value}`,
+        },
+    },
+];
+
+function getChartData(graduation: Graduation): UserSerie<Datum>[] {
+    const { ratingSystem: preferredSystem } = graduation;
+
+    const ratingHistory =
+        graduation.ratingHistories?.[preferredSystem]?.map(({ date, rating }) => ({
+            rating,
+            date: new Date(date),
+        })) ?? [];
+
+    return [{ label: 'Rating', data: ratingHistory }];
+}
+
+function StatLabel({ children, center }: { children: ReactNode; center?: boolean }) {
+    return (
+        <Typography
+            component='span'
+            fontSize='1rem'
+            variant='subtitle2'
+            color='text.secondary'
+            sx={{
+                textAlign: center ? 'center' : 'left',
```

Comment:
Are there any StatLabels that aren't centered? It seems like this prop could be removed and it's just always centered